btk/obs_conditions.py

```diff
@@ -3,18 +3,76 @@
 import numpy as np
 import galsim
 import descwl
+from collections import namedtuple
 
+# A simple class archetype to serve as a dictionary without having to write the field names every time
+Survey = namedtuple("Survey", ["name", "pixel_scale", "psf_scale", "bands", "mean_sky_level", "exp_time", "zero_point"])
```

Comment:
Those are a nice addition. I was thinking that I have not seen any checks on whether a given survey is appropriate for a catalog (ie you could not get HST images with Catsim and WLD for instance) ; should there be some implemented somewhere in the code ? (in the DrawBlendsGenerator maybe ?)

Reply:
That's right, we should make a check where not all surveys are compatible with all DrawBlendsGenerators. Maybe add a class attribute `compatible_surveys`?